[ 2022-05-02T12:11:31     INFO ] logging to applications insights
[ 2022-05-02T12:11:31     INFO ] [ exists                   ] True
[ 2022-05-02T12:11:32     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS', 'name': 'VMSSReImageRun1SOUTHCENTRALUS', 'type': 'Microsoft.Resources/resourceGroups', 'properties': <azure.mgmt.resource.resources.v2021_04_01.models._models_py3.ResourceGroupProperties object at 0x7fb5e1147d30>, 'location': 'southcentralus', 'managed_by': None, 'tags': None}
[ 2022-05-02T12:11:33     INFO ] [ create                   ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Storage/storageAccounts/vmsspipeline9fc2a6bc', 'name': 'vmsspipeline9fc2a6bc', 'type': 'Microsoft.Storage/storageAccounts', 'tags': {'confidentiality': 'public'}, 'location': 'southcentralus', 'sku': <azure.mgmt.storage.v2021_09_01.models._models_py3.Sku object at 0x7fb5e0f874c0>, 'kind': 'StorageV2', 'identity': None, 'extended_location': None, 'provisioning_state': 'Succeeded', 'primary_endpoints': <azure.mgmt.storage.v2021_09_01.models._models_py3.Endpoints object at 0x7fb5e0f87a60>, 'primary_location': 'southcentralus', 'status_of_primary': 'available', 'last_geo_failover_time': None, 'secondary_location': None, 'status_of_secondary': None, 'creation_time': datetime.datetime(2022, 4, 29, 16, 7, 7, 364576, tzinfo=<isodate.tzinfo.Utc object at 0x7fb5e4c8ee20>), 'custom_domain': None, 'sas_policy': None, 'key_policy': None, 'key_creation_time': <azure.mgmt.storage.v2021_09_01.models._models_py3.KeyCreationTime object at 0x7fb5e0f87c70>, 'secondary_endpoints': None, 'encryption': <azure.mgmt.storage.v2021_09_01.models._models_py3.Encryption object at 0x7fb5e0f87af0>, 'access_tier': 'Hot', 'azure_files_identity_based_authentication': None, 'enable_https_traffic_only': True, 'network_rule_set': <azure.mgmt.storage.v2021_09_01.models._models_py3.NetworkRuleSet object at 0x7fb5e0f87d90>, 'is_sftp_enabled': None, 'is_local_user_enabled': None, 'is_hns_enabled': None, 'geo_replication_stats': None, 'failover_in_progress': None, 'large_file_shares_state': None, 'private_endpoint_connections': [], 'routing_preference': None, 'blob_restore_status': None, 'allow_blob_public_access': True, 'minimum_tls_version': 'TLS1_0', 'allow_shared_key_access': None, 'enable_nfs_v3': None, 'allow_cross_tenant_replication': None, 'default_to_o_auth_authentication': None, 'public_network_access': None, 'immutable_storage_with_versioning': None, 'allowed_copy_scope': None, 'storage_account_sku_conversion_status': None, 'dns_endpoint_type': None}
[ 2022-05-02T12:11:34     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Network/networkSecurityGroups/vmssPeregrineTest-nsg', 'name': 'vmssPeregrineTest-nsg', 'type': 'Microsoft.Network/networkSecurityGroups', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'etag': 'W/"eb7d1979-ed01-4960-a05c-263594cf31d5"', 'security_rules': [<azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb5e08caa90>], 'default_security_rules': [<azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb5e08ca9a0>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb5e08cabe0>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb5e08cac10>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb5e08cac40>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb5e08cac70>, <azure.mgmt.network.v2021_05_01.models._models_py3.SecurityRule object at 0x7fb5e08caca0>], 'network_interfaces': None, 'subnets': [<azure.mgmt.network.v2021_05_01.models._models_py3.Subnet object at 0x7fb5e08cad00>], 'flow_logs': None, 'resource_guid': 'a66db6d6-770c-4f59-9ac2-c283de4a9a24', 'provisioning_state': 'Succeeded'}
[ 2022-05-02T12:11:35     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Network/virtualNetworks/vmssPeregrineTest-vnet', 'name': 'vmssPeregrineTest-vnet', 'type': 'Microsoft.Network/virtualNetworks', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'extended_location': None, 'etag': 'W/"d5b4eb7c-ea37-462b-9ea4-fbd232834c2c"', 'address_space': <azure.mgmt.network.v2021_05_01.models._models_py3.AddressSpace object at 0x7fb5e08cae50>, 'dhcp_options': None, 'flow_timeout_in_minutes': None, 'subnets': [<azure.mgmt.network.v2021_05_01.models._models_py3.Subnet object at 0x7fb5e0f87e50>], 'virtual_network_peerings': [], 'resource_guid': '0f1294d0-ba81-42f8-bd75-de499de66b2f', 'provisioning_state': 'Succeeded', 'enable_ddos_protection': False, 'enable_vm_protection': None, 'ddos_protection_plan': None, 'bgp_communities': None, 'encryption': None, 'ip_allocations': None}
[ 2022-05-02T12:11:35     INFO ] [ subnets_list             ] <iterator object azure.core.paging.ItemPaged at 0x7fb5e0f87f40>
[ 2022-05-02T12:11:36     INFO ] [ get                      ] {'additional_properties': {}, 'id': '/subscriptions/b698965a-1f53-45ec-8b81-58458d1b25f7/resourceGroups/VMSSReImageRun1SOUTHCENTRALUS/providers/Microsoft.Compute/virtualMachineScaleSets/vmssPeregrineTest', 'name': 'vmssPeregrineTest', 'type': 'Microsoft.Compute/virtualMachineScaleSets', 'location': 'southcentralus', 'tags': {'confidentiality': 'public'}, 'sku': <azure.mgmt.compute.v2021_11_01.models._models_py3.Sku object at 0x7fb5e063e790>, 'plan': None, 'identity': None, 'zones': ['1'], 'extended_location': None, 'upgrade_policy': None, 'automatic_repairs_policy': None, 'virtual_machine_profile': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineScaleSetVMProfile object at 0x7fb5e063eca0>, 'provisioning_state': 'Succeeded', 'overprovision': None, 'do_not_run_extensions_on_overprovisioned_v_ms': None, 'unique_id': '37746c0b-92b4-4b45-961f-474922163bb5', 'single_placement_group': False, 'zone_balance': None, 'platform_fault_domain_count': 1, 'proximity_placement_group': None, 'host_group': None, 'additional_capabilities': None, 'scale_in_policy': None, 'orchestration_mode': 'Flexible', 'spot_restore_policy': None, 'time_created': datetime.datetime(2022, 4, 29, 16, 7, 45, 135353, tzinfo=<FixedOffset '+00:00'>)}
[ 2022-05-02T12:11:36     INFO ] [ get_vms                  ] <iterator object azure.core.paging.ItemPaged at 0x7fb5e0901190>
[ 2022-05-02T12:11:36    DEBUG ] Using selector: EpollSelector
[ 2022-05-02T12:11:36     INFO ] [ get_vms                  ] <iterator object azure.core.paging.ItemPaged at 0x7fb5e05cca00>
[ 2022-05-02T12:11:36     INFO ] Finish queuing all vms
[ 2022-05-02T12:11:37     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance1OE4NY', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5e05f9d30>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5e05f9f10>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5e05f9dc0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5e05fd0a0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5e05fd0d0>], 'patch_status': None}
[ 2022-05-02T12:11:38     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance1ZB63N', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d37e0160>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d37e0340>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d37e01f0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d37e0490>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d37e04c0>], 'patch_status': None}
[ 2022-05-02T12:11:38     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance2C0PSP', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d37e7580>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d37e7760>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d37e7610>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d37e78b0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d37e78e0>], 'patch_status': None}
[ 2022-05-02T12:11:39     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance2RNIEP', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d37d5b20>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d37d5d00>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d37d5ac0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d37d5e80>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d37d5eb0>], 'patch_status': None}
[ 2022-05-02T12:11:39     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance2UT10B', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d37d5040>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2f7f190>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d37e0640>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f7f2e0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f7f310>], 'patch_status': None}
[ 2022-05-02T12:11:40     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance3BVHI3', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2f84550>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2f84730>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2f845e0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f84880>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f848b0>], 'patch_status': None}
[ 2022-05-02T12:11:40     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance3NM16Y', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2f98370>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2f98550>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2f98400>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f986a0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f986d0>], 'patch_status': None}
[ 2022-05-02T12:11:41     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance46BHVP', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2f8a760>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2f8d130>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2f8aa00>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f8d280>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f8d2b0>], 'patch_status': None}
[ 2022-05-02T12:11:42     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance4MZ3H7', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2fa64f0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2fa66d0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2fa6580>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2fa6820>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2fa6850>], 'patch_status': None}
[ 2022-05-02T12:11:42     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance4TUB0I', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2fad7f0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2fad9d0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2fad880>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2fadb20>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2fadb50>], 'patch_status': None}
[ 2022-05-02T12:11:42  WARNING ] AzureCliCredential.get_token failed: Unexpected output from Azure CLI: '--- Logging error ---
Traceback (most recent call last):
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/__main__.py", line 60, in <module>
    raise ex
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/__main__.py", line 53, in <module>
    sys.exit(exit_code)
SystemExit: 0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/handlers.py", line 70, in emit
    self.doRollover()
  File "/usr/lib/python3.8/logging/handlers.py", line 167, in doRollover
    os.rename(sfn, dfn)
FileNotFoundError: [Errno 2] No such file or directory: '/home/crodriguezde/.azure/telemetry/cache.6' -> '/home/crodriguezde/.azure/telemetry/cache.7'
Call stack:
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/__main__.py", line 123, in <module>
    telemetry.conclude()
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/core/telemetry.py", line 241, in _wrapper
    return func(*args, **kwargs)
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/core/decorators.py", line 79, in _wrapped_func
    return func(*args, **kwargs)
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/core/telemetry.py", line 297, in conclude
    save(get_config_dir(), _session.generate_payload())
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/telemetry/__init__.py", line 51, in save
    if save_payload(config_dir, payload) and should_upload(config_dir):
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/telemetry/util.py", line 53, in save_payload
    cache_saver.info(payload)
Message: '{"c4395b75-49cc-422c-bc95-c7d51aef5d46":[{"name":"azurecli/command","properties":{"Reserved.DataModel.EntityType":"UserTask","Reserved.DataModel.Action.Type":"Atomic","Reserved.DataModel.Action.Result":"Success","Reserved.ChannelUsed":"AI","Reserved.EventId":"12acd685-5e22-4501-8833-ce09a1dbaf2a","Reserved.SequenceNumber":1,"Reserved.SessionId":"4bed7af2a6cf57a6e33040d965e71390085a3771009d54cd20a82bcf53299f60","Reserved.TimeSinceSessionStart":0,"Reserved.DataModel.Source":"DataModelAPI","Reserved.DataModel.EntitySchemaVersion":4,"Reserved.DataModel.Severity":0,"Reserved.DataModel.ProductName":"azurecli","Reserved.DataModel.FeatureName":"command","Reserved.DataModel.EntityName":"account-getaccesstoken","Reserved.DataModel.CorrelationId":"689c12bd-0c68-4faa-a132-331d7283a6d6","Context.Default.VS.Core.ExeName":"azurecli","Context.Default.VS.Core.ExeVersion":"2.36.0@none","Context.Default.VS.Core.MacAddressHash":"e9bfd1348393ff8f258c5b795e82bf541f4459c5b4d6fa911e34d480ac33d348","Context.Default.VS.Core.Machine.Id":"e9bfd134-8393-ff8f-258c-5b795e82bf54","Context.Default.VS.Core.OS.Type":"linux","Context.Default.VS.Core.OS.Version":"#1 smp wed mar 2 00:30:59 utc 2022","Context.Default.VS.Core.OS.Platform":"linux-5.10.102.1-microsoft-standard-wsl2-x86_64-with-glibc2.29","Context.Default.VS.Core.User.Id":"64c5b0af-e344-11eb-b316-c03c59895c0a","Context.Default.VS.Core.User.IsMicrosoftInternal":"False","Context.Default.VS.Core.User.IsOptedIn":"True","Context.Default.VS.Core.TelemetryApi.ProductVersion":"azurecli@2.36.0","Context.Default.AzureCLI.Source":"az","Context.Default.AzureCLI.ClientRequestId":"ef36931c-ca3a-11ec-91cf-00155d785913","Context.Default.AzureCLI.CoreVersion":"2.36.0","Context.Default.AzureCLI.TelemetryVersion":"2.0","Context.Default.AzureCLI.InstallationId":"64c5b0af-e344-11eb-b316-c03c59895c0a","Context.Default.AzureCLI.ShellType":"/usr/bin/zsh","Context.Default.AzureCLI.UserAzureId":"39921e2f44771b3d76a90cf4cac837fdf8e646729bbbcbe017a174f2298b5c4c","Context.Default.AzureCLI.UserAzureSubscriptionId":"b698965a-1f53-45ec-8b81-58458d1b25f7","Context.Default.AzureCLI.DefaultOutputType":"unknown","Context.Default.AzureCLI.EnvironmentVariables":"[]","Context.Default.AzureCLI.Locale":"en_US,UTF-8","Context.Default.AzureCLI.StartTime":"2022-05-02 17:11:41.176610","Context.Default.AzureCLI.EndTime":"2022-05-02 17:11:41.319623","Context.Default.AzureCLI.InitTimeElapsed":"0.146489","Context.Default.AzureCLI.InvokeTimeElapsed":"0.140487","Context.Default.AzureCLI.OutputType":"json","Context.Default.AzureCLI.RawCommand":"account get-access-token","Context.Default.AzureCLI.Params":"--output,--resource","Context.Default.AzureCLI.PythonVersion":"3.8.10","Context.Default.AzureCLI.Mode":"default","Context.Default.AzureCLI.Installer":"PIP","Context.Default.AzureCLI.error_type":"None","Context.Default.AzureCLI.exception_name":"None","Context.Default.AzureCLI.debug_info":"","Context.Default.AzureCLI.PollStartTime":"None","Context.Default.AzureCLI.PollEndTime":"None"}}]}'
Arguments: ()
{
  ****,
  "expiresOn": "2022-05-02 12:54:18.000000",
  "subscription": "b698965a-1f53-45ec-8b81-58458d1b25f7",
  "tenant": "72f988bf-86f1-41af-91ab-2d7cd011db47",
  "tokenType": "Bearer"
}
'. 
To mitigate this issue, please refer to the troubleshooting guidelines here at https://aka.ms/azsdk/python/identity/azclicredential/troubleshoot.
[ 2022-05-02T12:11:43     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance6J1CJ8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2f4b9a0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2f4bb80>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2f4ba30>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f4bcd0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f4bd00>], 'patch_status': None}
[ 2022-05-02T12:11:43    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:11:43    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:11:43    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:11:43    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:11:43    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:11:43    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:11:43    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:11:43    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:11:43    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:11:44    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:06     INFO ] VMs in vm_queue:90
[ 2022-05-02T12:12:06     INFO ] VMs finish:0
[ 2022-05-02T12:12:14     INFO ] imds
[ 2022-05-02T12:12:14     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance7LN82P', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06ba880>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06baa60>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06ba910>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06babb0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06babe0>], 'patch_status': None}
[ 2022-05-02T12:12:15     INFO ] imds
[ 2022-05-02T12:12:15     INFO ] imds
[ 2022-05-02T12:12:15     INFO ] imds
[ 2022-05-02T12:12:15     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance7T6J4T', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0726d00>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0726640>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0726df0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0726760>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d07267c0>], 'patch_status': None}
[ 2022-05-02T12:12:16     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance8OF109', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06d89a0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06d8b80>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06d8a30>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06d8cd0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06d8d00>], 'patch_status': None}
[ 2022-05-02T12:12:16     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance8Y7YDL', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06dd7f0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06dd9d0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06dd880>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06ddb20>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06ddb50>], 'patch_status': None}
[ 2022-05-02T12:12:16     INFO ] imds
[ 2022-05-02T12:12:16     INFO ] imds
[ 2022-05-02T12:12:16     INFO ] imds
[ 2022-05-02T12:12:17     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance91S4YT', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d37f5610>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0709700>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d37e0b50>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0709550>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d07099a0>], 'patch_status': None}
[ 2022-05-02T12:12:18     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstance9PSKLN', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06e0880>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06e0a60>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06e0910>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06e0bb0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06e0be0>], 'patch_status': None}
[ 2022-05-02T12:12:18     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceAIQZ3V', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06ef730>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06ef910>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06ef7c0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06efa60>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06efa90>], 'patch_status': None}
[ 2022-05-02T12:12:18     INFO ] imds
[ 2022-05-02T12:12:18     INFO ] imds
[ 2022-05-02T12:12:19     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceBEHG6E', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06f95b0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06f9790>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06f9640>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06f98e0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06f9910>], 'patch_status': None}
[ 2022-05-02T12:12:20     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceBWHIYH', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06800a0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0680280>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0680130>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06803d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0680400>], 'patch_status': None}
[ 2022-05-02T12:12:20     INFO ] imds
[ 2022-05-02T12:12:20     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceC61RZ6', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0696130>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0696310>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06961c0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0696460>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0696490>], 'patch_status': None}
[ 2022-05-02T12:12:20    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:20    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:20    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:20    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:20    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:20    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:20    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:20    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:21    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:21    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:36     INFO ] VMs in vm_queue:80
[ 2022-05-02T12:12:36     INFO ] VMs finish:10
[ 2022-05-02T12:12:51     INFO ] imds
[ 2022-05-02T12:12:51     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceGD1WLL', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2f98f10>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2f48940>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2f988b0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f48700>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f48730>], 'patch_status': None}
[ 2022-05-02T12:12:51     INFO ] imds
[ 2022-05-02T12:12:51     INFO ] imds
[ 2022-05-02T12:12:51     INFO ] imds
[ 2022-05-02T12:12:51     INFO ] imds
[ 2022-05-02T12:12:51     INFO ] imds
[ 2022-05-02T12:12:52     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceGNDTJV', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06dd430>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2fade50>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06dd550>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2fad1c0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2fad220>], 'patch_status': None}
[ 2022-05-02T12:12:53     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceGQONHL', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2f84910>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2f7a8e0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2f7a130>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f8d970>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f8de50>], 'patch_status': None}
[ 2022-05-02T12:12:53     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceIEPJB8', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2f556d0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2f55700>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2f557f0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f550a0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f55af0>], 'patch_status': None}
[ 2022-05-02T12:12:54     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceII52NC', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0686970>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0696640>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06964f0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0696790>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06967c0>], 'patch_status': None}
[ 2022-05-02T12:12:55     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceJ1DD4T', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0696d00>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0696ee0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0696d90>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0640070>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06400a0>], 'patch_status': None}
[ 2022-05-02T12:12:55     INFO ] imds
[ 2022-05-02T12:12:56     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceK8QFGE', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0640be0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0640dc0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0640c70>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0640f10>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0640f40>], 'patch_status': None}
[ 2022-05-02T12:12:56     INFO ] imds
[ 2022-05-02T12:12:56     INFO ] imds
[ 2022-05-02T12:12:56     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceKXL20R', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0642940>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0642b20>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06429d0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0642c70>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0642ca0>], 'patch_status': None}
[ 2022-05-02T12:12:57     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceLDI2O5', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0644700>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06448e0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0644460>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0644ac0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0644af0>], 'patch_status': None}
[ 2022-05-02T12:12:57  WARNING ] AzureCliCredential.get_token failed: Unexpected output from Azure CLI: '--- Logging error ---
Traceback (most recent call last):
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/__main__.py", line 60, in <module>
    raise ex
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/__main__.py", line 53, in <module>
    sys.exit(exit_code)
SystemExit: 0

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/logging/handlers.py", line 70, in emit
    self.doRollover()
  File "/usr/lib/python3.8/logging/handlers.py", line 167, in doRollover
    os.rename(sfn, dfn)
FileNotFoundError: [Errno 2] No such file or directory: '/home/crodriguezde/.azure/telemetry/cache.7' -> '/home/crodriguezde/.azure/telemetry/cache.8'
Call stack:
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/__main__.py", line 123, in <module>
    telemetry.conclude()
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/core/telemetry.py", line 241, in _wrapper
    return func(*args, **kwargs)
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/core/decorators.py", line 79, in _wrapped_func
    return func(*args, **kwargs)
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/core/telemetry.py", line 297, in conclude
    save(get_config_dir(), _session.generate_payload())
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/telemetry/__init__.py", line 51, in save
    if save_payload(config_dir, payload) and should_upload(config_dir):
  File "/home/crodriguezde/.local/lib/python3.8/site-packages/azure/cli/telemetry/util.py", line 53, in save_payload
    cache_saver.info(payload)
Message: '{"c4395b75-49cc-422c-bc95-c7d51aef5d46":[{"name":"azurecli/command","properties":{"Reserved.DataModel.EntityType":"UserTask","Reserved.DataModel.Action.Type":"Atomic","Reserved.DataModel.Action.Result":"Success","Reserved.ChannelUsed":"AI","Reserved.EventId":"30f9901c-b3ad-456e-8b62-edb0532e83e6","Reserved.SequenceNumber":1,"Reserved.SessionId":"4bed7af2a6cf57a6e33040d965e71390085a3771009d54cd20a82bcf53299f60","Reserved.TimeSinceSessionStart":0,"Reserved.DataModel.Source":"DataModelAPI","Reserved.DataModel.EntitySchemaVersion":4,"Reserved.DataModel.Severity":0,"Reserved.DataModel.ProductName":"azurecli","Reserved.DataModel.FeatureName":"command","Reserved.DataModel.EntityName":"account-getaccesstoken","Reserved.DataModel.CorrelationId":"6b81e792-d67b-421c-bcb2-dbbaf5bd882f","Context.Default.VS.Core.ExeName":"azurecli","Context.Default.VS.Core.ExeVersion":"2.36.0@none","Context.Default.VS.Core.MacAddressHash":"e9bfd1348393ff8f258c5b795e82bf541f4459c5b4d6fa911e34d480ac33d348","Context.Default.VS.Core.Machine.Id":"e9bfd134-8393-ff8f-258c-5b795e82bf54","Context.Default.VS.Core.OS.Type":"linux","Context.Default.VS.Core.OS.Version":"#1 smp wed mar 2 00:30:59 utc 2022","Context.Default.VS.Core.OS.Platform":"linux-5.10.102.1-microsoft-standard-wsl2-x86_64-with-glibc2.29","Context.Default.VS.Core.User.Id":"64c5b0af-e344-11eb-b316-c03c59895c0a","Context.Default.VS.Core.User.IsMicrosoftInternal":"False","Context.Default.VS.Core.User.IsOptedIn":"True","Context.Default.VS.Core.TelemetryApi.ProductVersion":"azurecli@2.36.0","Context.Default.AzureCLI.Source":"az","Context.Default.AzureCLI.ClientRequestId":"1c611f7e-ca3b-11ec-b901-00155d785913","Context.Default.AzureCLI.CoreVersion":"2.36.0","Context.Default.AzureCLI.TelemetryVersion":"2.0","Context.Default.AzureCLI.InstallationId":"64c5b0af-e344-11eb-b316-c03c59895c0a","Context.Default.AzureCLI.ShellType":"/usr/bin/zsh","Context.Default.AzureCLI.UserAzureId":"39921e2f44771b3d76a90cf4cac837fdf8e646729bbbcbe017a174f2298b5c4c","Context.Default.AzureCLI.UserAzureSubscriptionId":"b698965a-1f53-45ec-8b81-58458d1b25f7","Context.Default.AzureCLI.DefaultOutputType":"unknown","Context.Default.AzureCLI.EnvironmentVariables":"[]","Context.Default.AzureCLI.Locale":"en_US,UTF-8","Context.Default.AzureCLI.StartTime":"2022-05-02 17:12:56.952952","Context.Default.AzureCLI.EndTime":"2022-05-02 17:12:57.145928","Context.Default.AzureCLI.InitTimeElapsed":"0.131614","Context.Default.AzureCLI.InvokeTimeElapsed":"0.186824","Context.Default.AzureCLI.OutputType":"json","Context.Default.AzureCLI.RawCommand":"account get-access-token","Context.Default.AzureCLI.Params":"--output,--resource","Context.Default.AzureCLI.PythonVersion":"3.8.10","Context.Default.AzureCLI.Mode":"default","Context.Default.AzureCLI.Installer":"PIP","Context.Default.AzureCLI.error_type":"None","Context.Default.AzureCLI.exception_name":"None","Context.Default.AzureCLI.debug_info":"","Context.Default.AzureCLI.PollStartTime":"None","Context.Default.AzureCLI.PollEndTime":"None"}}]}'
Arguments: ()
{
  ****,
  "expiresOn": "2022-05-02 12:54:18.000000",
  "subscription": "b698965a-1f53-45ec-8b81-58458d1b25f7",
  "tenant": "72f988bf-86f1-41af-91ab-2d7cd011db47",
  "tokenType": "Bearer"
}
'. 
To mitigate this issue, please refer to the troubleshooting guidelines here at https://aka.ms/azsdk/python/identity/azclicredential/troubleshoot.
[ 2022-05-02T12:12:58     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceLQMDV3', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0649430>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0649610>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06494c0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0649760>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0649790>], 'patch_status': None}
[ 2022-05-02T12:12:58     INFO ] imds
[ 2022-05-02T12:12:59     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceLU5LK6', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0652d60>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0652f40>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0652df0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06590d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0659100>], 'patch_status': None}
[ 2022-05-02T12:12:59    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:59    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:59    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:59    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:59    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:59    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:59    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:59    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:59    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:12:59    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:13:06     INFO ] VMs in vm_queue:70
[ 2022-05-02T12:13:06     INFO ] VMs finish:20
[ 2022-05-02T12:13:29     INFO ] imds
[ 2022-05-02T12:13:29     INFO ] imds
[ 2022-05-02T12:13:30     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceNA9EB5', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06c3370>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06c3220>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06c3d30>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d37d3cd0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d37d3f10>], 'patch_status': None}
[ 2022-05-02T12:13:30     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceO2QE0I', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2f8ac10>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2f8aa00>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2f8aac0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f8a730>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f8a430>], 'patch_status': None}
[ 2022-05-02T12:13:30     INFO ] imds
[ 2022-05-02T12:13:31     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceP2C23I', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d071c340>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d071c7f0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d071cd30>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d071c4f0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d071c310>], 'patch_status': None}
[ 2022-05-02T12:13:31     INFO ] imds
[ 2022-05-02T12:13:31     INFO ] imds
[ 2022-05-02T12:13:31     INFO ] imds
[ 2022-05-02T12:13:32     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstancePTYO8I', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0726370>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2fa6850>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5e05ce670>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06d8490>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f7fc40>], 'patch_status': None}
[ 2022-05-02T12:13:32     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceQ6FKSE', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2f84100>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2f84bb0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2f84580>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d071c220>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d071c160>], 'patch_status': None}
[ 2022-05-02T12:13:33     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceQRBUZY', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d37f5370>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d37f51c0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d37f5af0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06525b0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06528b0>], 'patch_status': None}
[ 2022-05-02T12:13:33     INFO ] imds
[ 2022-05-02T12:13:33     INFO ] imds
[ 2022-05-02T12:13:34     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceSHLEHR', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0643460>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06ef1c0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06433a0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06efe50>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06effa0>], 'patch_status': None}
[ 2022-05-02T12:13:35     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceSXXXM3', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06608e0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0660ac0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0660970>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0660c10>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0660c40>], 'patch_status': None}
[ 2022-05-02T12:13:35     INFO ] imds
[ 2022-05-02T12:13:35     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceTEOLOE', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06026d0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06028b0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0602760>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0602a00>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0602a30>], 'patch_status': None}
[ 2022-05-02T12:13:36     INFO ] imds
[ 2022-05-02T12:13:36     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceUM6OHC', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0680400>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06effd0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0683430>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0686400>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0686460>], 'patch_status': None}
[ 2022-05-02T12:13:36    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:13:36    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:13:36    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:13:36    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:13:36    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:13:36    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:13:36    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:13:36    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:13:36     INFO ] VMs in vm_queue:60
[ 2022-05-02T12:13:36     INFO ] VMs finish:30
[ 2022-05-02T12:13:36    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:13:37    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:14:06     INFO ] VMs in vm_queue:60
[ 2022-05-02T12:14:06     INFO ] VMs finish:30
[ 2022-05-02T12:14:07     INFO ] imds
[ 2022-05-02T12:14:07     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceUS29IZ', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0659490>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06424f0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0659ac0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0642f40>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06428e0>], 'patch_status': None}
[ 2022-05-02T12:14:07     INFO ] imds
[ 2022-05-02T12:14:07     INFO ] imds
[ 2022-05-02T12:14:07     INFO ] imds
[ 2022-05-02T12:14:07     INFO ] imds
[ 2022-05-02T12:14:07     INFO ] imds
[ 2022-05-02T12:14:08     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceUT8N78', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d0644400>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0696970>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d071cee0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f5b9d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f5ba30>], 'patch_status': None}
[ 2022-05-02T12:14:09     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceV09E6G', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2f8d5b0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2f8d550>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2f8dc40>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f8d430>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2fad040>], 'patch_status': None}
[ 2022-05-02T12:14:09     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceY49PZP', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d070b160>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d070b6d0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d070b430>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f7a6d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f7a700>], 'patch_status': None}
[ 2022-05-02T12:14:10     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceYJ1UJU', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d38deb20>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d38de040>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d38de910>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06ba760>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06ba430>], 'patch_status': None}
[ 2022-05-02T12:14:11     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceYPWCON', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d2fb09d0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d2fb0340>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d2fb0760>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2fb0e20>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2fb0f70>], 'patch_status': None}
[ 2022-05-02T12:14:11     INFO ] imds
[ 2022-05-02T12:14:11     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceYTBEAJ', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d07266a0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d07260a0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0726e50>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06fd0d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06fd100>], 'patch_status': None}
[ 2022-05-02T12:14:12     INFO ] imds
[ 2022-05-02T12:14:12     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceYEQ12P', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d37e72e0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0660c40>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d37e7be0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06609d0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0660a30>], 'patch_status': None}
[ 2022-05-02T12:14:12     INFO ] imds
[ 2022-05-02T12:14:13     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceXVOAZU', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06605b0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d0660fd0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d0660e80>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0643070>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d0643310>], 'patch_status': None}
[ 2022-05-02T12:14:13     INFO ] imds
[ 2022-05-02T12:14:14     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceXAVI7R', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d061c760>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d061c940>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d061c7f0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d061ca90>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d061cac0>], 'patch_status': None}
[ 2022-05-02T12:14:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:14:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:14:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:14:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:14:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:14:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:14:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:14:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:14:14    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:14:15    DEBUG ] Encoding detection on empty bytes, assuming utf_8 intention.
[ 2022-05-02T12:14:36     INFO ] VMs in vm_queue:50
[ 2022-05-02T12:14:36     INFO ] VMs finish:40
[ 2022-05-02T12:14:44     INFO ] imds
[ 2022-05-02T12:14:45     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceWRL3N7', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06ddfa0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06dd490>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06dd1c0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06dd2e0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d06dd580>], 'patch_status': None}
[ 2022-05-02T12:14:45     INFO ] imds
[ 2022-05-02T12:14:45     INFO ] imds
[ 2022-05-02T12:14:45     INFO ] imds
[ 2022-05-02T12:14:45     INFO ] imds
[ 2022-05-02T12:14:45     INFO ] imds
[ 2022-05-02T12:14:46     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceWQT136', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06ba5e0>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d06baf10>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d06bae50>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f8af70>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d2f8a4c0>], 'patch_status': None}
[ 2022-05-02T12:14:46     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceVO9WXJ', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d06c3820>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5d37d3fd0>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d37d3ca0>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d37d3eb0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5d37d3ee0>], 'patch_status': None}
[ 2022-05-02T12:14:47     INFO ] [ get                      ] {'additional_properties': {}, 'platform_update_domain': None, 'platform_fault_domain': None, 'computer_name': 'vmInstanceUC5FPI', 'os_name': 'ubuntu', 'os_version': '18.04', 'hyper_v_generation': 'V1', 'rdp_thumb_print': None, 'vm_agent': <azure.mgmt.compute.v2021_11_01.models._models_py3.VirtualMachineAgentInstanceView object at 0x7fb5d070bc10>, 'maintenance_redeploy_status': None, 'disks': [<azure.mgmt.compute.v2021_11_01.models._models_py3.DiskInstanceView object at 0x7fb5e05ce520>], 'extensions': [], 'vm_health': None, 'boot_diagnostics': <azure.mgmt.compute.v2021_11_01.models._models_py3.BootDiagnosticsInstanceView object at 0x7fb5d070b850>, 'assigned_host': None, 'statuses': [<azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5e05ce4f0>, <azure.mgmt.compute.v2021_11_01.models._models_py3.InstanceViewStatus object at 0x7fb5e05ce5e0>], 'patch_status': None}
[ 2022-05-02T12:14:49    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7fb5d2fadc40>
[ 2022-05-02T12:14:49    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7fb5d06d8ca0>
[ 2022-05-02T12:14:49    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7fb5d2fad310>
[ 2022-05-02T12:14:49    ERROR ] Unclosed client session
client_session: <aiohttp.client.ClientSession object at 0x7fb5d2f7a850>
[ 2022-05-02T12:14:49     INFO ] Exit per user request
[ 2022-05-02T12:14:50    ERROR ] Task exception was never retrieved
future: <Task finished name='Task-14' coro=<_get() done, defined at /home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/backoff/_async.py:118> exception=KeyboardInterrupt()>
Traceback (most recent call last):
  File "deployer.py", line 150, in <module>
    main()
  File "deployer.py", line 137, in main
    asyncio.run(imds.get_from_vmss_vms(args))
  File "/usr/lib/python3.8/asyncio/runners.py", line 44, in run
    return loop.run_until_complete(main)
  File "/usr/lib/python3.8/asyncio/base_events.py", line 603, in run_until_complete
    self.run_forever()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 570, in run_forever
    self._run_once()
  File "/usr/lib/python3.8/asyncio/base_events.py", line 1859, in _run_once
    handle._run()
  File "/usr/lib/python3.8/asyncio/events.py", line 81, in _run
    self._context.run(self._callback, *self._args)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/backoff/_async.py", line 133, in retry
    ret = await target(*args, **kwargs)
  File "/home/crodriguezde/experiments/reimage/imds.py", line 51, in _get
    instance_vm = _wait_vm_state_running(cli_args, vmss_vm)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/backoff/_sync.py", line 94, in retry
    ret = target(*args, **kwargs)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/backoff/_sync.py", line 43, in retry
    ret = target(*args, **kwargs)
  File "/home/crodriguezde/experiments/reimage/imds.py", line 93, in _wait_vm_state_running
    instance_vm = vm.get(args, vmss_vm)
  File "/home/crodriguezde/experiments/reimage/utils.py", line 14, in timer
    result = fn(*args, **kwargs)
  File "/home/crodriguezde/experiments/reimage/vm.py", line 31, in get
    return resource_client.virtual_machines.instance_view(
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/tracing/decorator.py", line 73, in wrapper_use_tracer
    return func(*args, **kwargs)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/mgmt/compute/v2021_11_01/operations/_virtual_machines_operations.py", line 1626, in instance_view
    pipeline_response = self._client._pipeline.run(request, stream=False, **kwargs)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/_base.py", line 211, in run
    return first_node.send(pipeline_request)  # type: ignore
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/_base.py", line 71, in send
    response = self.next.send(request)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/_base.py", line 71, in send
    response = self.next.send(request)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/_base.py", line 71, in send
    response = self.next.send(request)
  [Previous line repeated 2 more times]
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/mgmt/core/policies/_base.py", line 47, in send
    response = self.next.send(request)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/policies/_redirect.py", line 158, in send
    response = self.next.send(request)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/policies/_retry.py", line 445, in send
    response = self.next.send(request)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/policies/_authentication.py", line 119, in send
    response = self.next.send(request)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/_base.py", line 71, in send
    response = self.next.send(request)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/_base.py", line 71, in send
    response = self.next.send(request)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/_base.py", line 71, in send
    response = self.next.send(request)
  [Previous line repeated 1 more time]
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/_base.py", line 103, in send
    self._sender.send(request.http_request, **request.context.options),
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/azure/core/pipeline/transport/_requests_basic.py", line 327, in send
    response = self.session.request(  # type: ignore
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/requests/sessions.py", line 529, in request
    resp = self.send(prep, **send_kwargs)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/requests/sessions.py", line 645, in send
    r = adapter.send(request, **kwargs)
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/requests/adapters.py", line 440, in send
    resp = conn.urlopen(
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/urllib3/connectionpool.py", line 703, in urlopen
    httplib_response = self._make_request(
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/urllib3/connectionpool.py", line 449, in _make_request
    six.raise_from(e, None)
  File "<string>", line 3, in raise_from
  File "/home/crodriguezde/experiments/reimage/venv/lib/python3.8/site-packages/urllib3/connectionpool.py", line 444, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/lib/python3.8/http/client.py", line 1348, in getresponse
    response.begin()
  File "/usr/lib/python3.8/http/client.py", line 316, in begin
    version, status, reason = self._read_status()
  File "/usr/lib/python3.8/http/client.py", line 277, in _read_status
    line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
  File "/usr/lib/python3.8/socket.py", line 669, in readinto
    return self._sock.recv_into(b)
  File "/usr/lib/python3.8/ssl.py", line 1241, in recv_into
    return self.read(nbytes, buffer)
  File "/usr/lib/python3.8/ssl.py", line 1099, in read
    return self._sslobj.read(len, buffer)
KeyboardInterrupt
